state/runtime/evm/instructions.go

```diff
@@ -718,18 +652,22 @@ func opExtCodeSize(c *state) {
 		return
 	}
 
-	c.push1().SetUint64(uint64(c.host.GetCodeSize(addr)))
+	x := uint256.NewInt(uint64(c.host.GetCodeSize(addr)))
+	c.push(*x)
 }
 
 func opGasPrice(c *state) {
-	c.push1().SetBytes(c.host.GetTxContext().GasPrice.Bytes())
+	x := uint256.NewInt(0)
+	x.SetBytes(c.host.GetTxContext().GasPrice.Bytes())
+	c.push(*x)
```

Comment:
Is this suboptimal, because we are using value typed reference, that gets pushed to the stack (instead to be placed on the heap)? Or is it not important at all? Dereferencing of `x` made me to think in this direction...
```suggestion
	x := uint256.Int{0}
	x.SetBytes(c.host.GetTxContext().GasPrice.Bytes())
	c.push(x)
```